applications/DEMApplication/tests/test_glued_particles.py

```diff
@@ -54,13 +57,6 @@ def test_Glued_Particles_1(self):
         model = Kratos.Model()
         auxiliary_functions_for_tests.CreateAndRunStageInSelectedNumberOfOpenMPThreads(GluedParticlesTestSolution, model, parameters_file_name, 1)
```

Comment:
Is it possible to change this 1 by 
`auxiliary_functions_for_tests.GetHardcodedNumberOfThreads()`
?

Reply:
if it works, we can use it for all tests. So changing one number in one file, all of them change the behaviour.

```
def  GetHardcodedNumberOfThreads():
    return 1
```